### PR TITLE
Archana_Assignment_25_10min_Done

### PR DESCRIPTION
**Pull request by ArchanaNJun20**
image

### CODE REVIEW

**Review by AavrutiDJun20 — APPROVED** [review]
Code looks good to me

File: src/archanaNJun20/SumOfDigit.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+public class SumOfDigit {
+	void printSumOfDigit(String input){
+		int sum=0;
+		for(int index=0;index<input.length();index++){
+			if(Character.isDigit(input.charAt(index))){
+				String charactertostring=String.valueOf(input.charAt(index));
+				int stringtointeger=Integer.parseInt(charactertostring);
+				sum+=stringtointeger;
+			}
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/SumOfDigit.java:11]: You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**